src/Http/Routing/src/RouteOptions.cs

```diff
@@ -72,6 +72,7 @@ internal ICollection<EndpointDataSource> EndpointDataSources
         /// </summary>
         public IDictionary<string, Type> ConstraintMap
         {
+            [RequiresUnreferencedCode($"The linker cannot determine what constraints are being added via the ConstraintMap property. Prefer {nameof(RouteOptions)}.{nameof(SetParameterPolicy)} instead for setting constraints. This warning can be suppressed if this property is being used to read of delete constraints.")]
```

Comment:
Don't say `linker` in public facing docs/strings. Instead, we prefer the term `trimmer` or `trimming`.

> is being used to read of delete constraints.

I'm not sure I understand what that means. Is there a type-o or missing word?

Reply:
ah hah